src/main/resources/view/ReacheTheme.css

```diff
@@ -1,34 +1,40 @@
 .root {
-    -darkgrey: #383838;
-    -yellow: #f2d16a;
-    -lightgrey: #e9e9e9;
-    -lightblue: #75c7fb;
-    -blue: #0048cd;
+    -primary: #0048cd;
+    -primary_dark: #00379c;
+    -primary_light: #75c7fb;
+    -secondary: #ffcf00;
+    -accent: #ecf1f4;
+    -border: #d4dae6;
+    -background_primary: #fafcfe;
+    -background_secondary: #f4f6f9;
+    -text_primary: #404040;
+    -text_secondary: #555555;
+    -grey: #686F7C;
 }
```

Comment:
New UI looks great! Can you add the same theme colours in the ConfirmWindow and HelpWindow as well?

Reply:
Ah thats a good idea I'll do that